properties.schema

```diff
@@ -137,6 +137,15 @@
                   "help": "Controls whether the assessment should be reset automatically (up to the number of available attempts) when a user revisits the page.",
                   "validators": []
                 },
+                "_isResetIfFailed": {
+                  "type": "boolean",
+                  "required": true,
+                  "default": false,
```

Comment:
Should this be `true`?  That would make it consistent with line 22 of adapt-assessmentArticleModel.js and in line with current default behaviour.

Reply:
I'm not that fussed either way, however it is a tad surprising that it hard resets on fail when it doesn't on partial completion or pass. Maybe that is just me though. Happy to switch it.

Reply:
@davetaz what @brian-learningpool  is saying is that you have defaulted `_isResetIfFailed` to `true` in adapt-assessmentArticleModel.js but defaulted it to `false` in properties.schema, which means AT users will get different default behaviour to framework users.

Reply:
This is now updated to match legacy behaviour

Reply:
@davetaz you've changed `_isResetOnRevisit` but @brian-learningpool was actually referring to the new property you have added: `_isResetIfFailed` which you have set to default to `true` for Framework users but `false` for Authoring Tool users, please could you change the entry for this property in properties.schema so that it defaults to `false` (and revert the changes you made to `_isResetOnRevisit`)?

Reply:
although before you do anything https://gitter.im/adaptlearning/adapt_framework?at=5b4e16e203a5fa639e60eefc